app/.env.example

```diff
@@ -13,6 +13,9 @@
 # https://www.prisma.io/docs/reference/database-reference/connection-urls#env
 DATABASE_URL="postgresql://postgres:postgres@localhost:5432/openpipe?schema=public"
 
+# Used by docker containers to request training data from the API
+DOCKER_SECRET="placeholder"
```

Comment:
This isn't a great name. For one thing, all our software runs in Docker. Maybe AUTHENTICATED_SYSTEM_KEY?